lib/matplotlib/backends/backend_agg.py

```diff
@@ -448,6 +448,9 @@ def buffer_rgba(self):
         return self.renderer.buffer_rgba()
 
     def print_raw(self, filename_or_obj, *args, **kwargs):
+        self._check_savefig_extra_args(kwargs)
+        del kwargs
```

Comment:
perhaps turn this into a decorator so that you can write
```python
@_check_savefig_extra_args
def print_raw(self, filename_or_obj, *args):
```
and not have to `del kwargs`?

Reply:
Unfortunately, it's not always called immediately at the start of a function, as sometimes they do really process `kwargs`. I threw in the `del kwargs` to be sure it was really not used after, though it's not strictly required.

I suppose I can try to make some of those remaining `kwargs` explicit instead, and then a decorator would work.

Reply:
Turns out there was only one remaining case like that, so that was not as difficult to do as I expected.